plugins/scaffolder-react/src/next/components/TemplateCard/CardHeader.tsx

```diff
@@ -36,6 +36,9 @@ const useStyles = makeStyles<
     display: 'flex',
     justifyContent: 'space-between',
   },
+  detailIcon: {
+    color: ({ cardFontColor }) => cardFontColor,
```

Comment:
Wasn't it enough to put an `inherit` color here? Or, perhaps, inside the `TemplateDetailButton` component itself instead if possible. The favourite star next to this icon doesn't have to do any trickery I think?

Reply:
I tried to find a root cause of different color, without success. It's probably taken from <Link>. I didn't find any other difference in usage this icon and Favorite star otherwise. Only this solution worked for me.

Reply:
➕ I tried to take a swing at this a well and it wasn't a simple task. Agree it looks to be from the Link.